File: Projects/Metanol Screen Saver/AssemblyStage.vb
Imports Tools.InternalT

'This file should be shared with all VB project in Tools
#If Config = Nightly Then
<Assembly: AssemblyBuildStage(BuildStates.Nightly)> 
#ElseIf Config = Alpha Then
<Assembly: AssemblyBuildStage(BuildStates.Alpha)>
#ElseIf Config = Beta Then
<Assembly: AssemblyBuildStage(BuildStates.Beta)>
#ElseIf Config = RC Then
<Assembly: AssemblyBuildStage(BuildStates.RC)>
#ElseIf Config = Release Then
<Assembly: AssemblyBuildStage(BuildStates.Release )>
#End If

File: Tools VisualStudio CS/AssemblyStage.cs
//This file should be shared by all C# project in ĐTools
using Tools.InternalT;
#if Release
[assembly: AssemblyBuildStageAttribute(BuildStates.Release)]
#else
#if RC
[assembly: AssemblyBuildStageAttribute(BuildStates.RC)]
#else
#if Beta
[assembly: AssemblyBuildStageAttribute(BuildStates.Beta)]
#else
#if Alpha
[assembly: AssemblyBuildStageAttribute(BuildStates.Alpha)]
#else
#if Nightly
[assembly: AssemblyBuildStageAttribute(BuildStates.Nightly)]
#endif
#endif
#endif
#endif
#endif 